src/lib/Components/NavigationHub.php

```diff
@@ -33,15 +33,20 @@ class NavigationHub implements Component
      */
     protected $request;
 
-    public function __construct(EngineInterface $templating, Request $request, array $zones = [], array $links = [])
+    public function __construct(EngineInterface $templating, Request $request = null, array $zones = [], array $links = [])
```

Comment:
Yes, `Fixed an error with Request when using debug:event-dispatcher`. CLI doesn't like the Request as a required service ctor argument.